ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -347,6 +351,7 @@ func (scbuilder *SiteConfigBuilder) getExtraManifest(dataMap map[string]interfac
 				}
 				if value != "" {
 					dataMap[filename] = value
+					doNotMerge[filename] = true
```

Comment:
It's probably worth a comment describing why we're excluding all templated MCs: just because the only templated MC is an installation-only MC.